### ISSUE TITLE
Buchhaltung Kontoabgleich AGE-Rechnungen

### ISSUE THREAD

**Issue by Toelk-A** [chrbusch/GTM-V#497]
AGE-Rechnungen sind in diesem Sinne keine Rechnung sondern eine Auflistung der durchgeführten Gewebespenden in einer meldenden Einrichtung/Bestatter/Pathologie.

Der GTM-V wird also ein Betrag "X" aus der AGE Auflistung in Rechnung gestellt. Ist dieser Betrag "X" durch die GTM-V an die genannte Einrichtung überwiesen worden, so können die AGE Auflistungen in den Kontobewegungsliste zugeordnet werden:

![image](https://user-images.githubusercontent.com/22314271/53553218-6da3b580-3b3d-11e9-88dd-b1267407cf82.png)

![image](https://user-images.githubusercontent.com/22314271/53553196-5e246c80-3b3d-11e9-9288-91f41674f372.png)

-> Zuordnung funktioniert, Auflistung /"Rechnung" wird auf bezahlt gesetzt. Allerdings wird die Kontobewegung nach Zuordnung auf "gelb" anstatt "grün" gesetzt. Hat dies einen Grund?

**Comment by mprueter** [comment]
ja, hat es. Bisher haben wir die AGE-Rechnungen hier außen vor gelassen und nur Eingänge verbucht und zugewiesen. Hier handelt es sich aber um einen Abgang vom Konto der GTMV. Die AGE-Rng werden im System aber wie Debitorrechnungen behandelt, also mit positiven Zahlen. Deswegen ist die Summe der zugeordneten Kontobewegung ungleich der Rechnungssumme und deswegen ist die Ampel gelb. 

![bildschirmfoto 2019-02-28 um 10 04 14](https://user-images.githubusercontent.com/3273572/53554889-13a4ef00-3b41-11e9-8d70-96a700098e3a.png)

**Comment by Toelk-A** [comment]
also eine weitere Baustelle zum Thema Buchhaltung.... ;-) Wir müssten also quasi eine Kreditorenrechnung hieraus erstellen..

**Comment by mprueter** [comment]
Das wäre die saubere Lösung. 
...oder wir machen beim Kontoabgleich eine Sonderregelung bei AGE-Rng.

**Comment by Toelk-A** [comment]
Ich glaube es ist an der Zeit "saubere Lösungen" zu finden. Je nach Zeitaufwand würde ich das Thema hinten anstellen, da wir erst einmal noch andere Baustellen haben. Wie viel Zeit würde eine Kreditorrechnungsprogrammierung in etwa in Anspruch nehmen?? Ich denke dort hängt auch das gesamt "AGE Abrechnungsskript" dran richtig?

**Comment by mprueter** [comment]
Da muss ich mich erstmal schlau machen...
